paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/ui/server_side_confirm/complete_flow/ServerSideConfirmationCompleteFlowViewModel.kt

```diff
@@ -91,10 +92,13 @@ internal class ServerSideConfirmationCompleteFlowViewModel(
                 CreateIntentResult.Success(apiResult.value.clientSecret)
             }
             is ApiResult.Failure -> {
+                val error = ExampleCreateAndConfirmErrorResponse.deserialize(
```

Comment:
I'm convinced this will throw exceptions in the case of a response that isn't json, or one that breaks at the socket layer. It would be nice to have tests for this.

Reply:
I added a try/catch, do you think this is sufficient for this example application?